service/frontend/workflowHandler.go

```diff
@@ -2490,6 +2492,7 @@ func (wh *WorkflowHandler) RequestCancelWorkflowExecution(
 		CancelRequest: cancelRequest,
 	})
 	if err != nil {
+		err = wh.NormalizeVersionedErrors(ctx, err)
```

Comment:
Could we do normalize after the wh.error so that we have the correct error metrics?